upgrade_tests/helpers/variants.py

```diff
@@ -7,71 +7,51 @@ class VersionError(Exception):
-to_ver = os.environ.get('TO_VERSION')
+_supported_ver = ['6.1', '6.2']
+_FROM_VERSION = os.environ.get('FROM_VERSION')
+_TO_VER = os.environ.get('TO_VERSION')
 
-if from_ver not in supported_ver:
+if _FROM_VERSION not in _supported_ver:
     raise VersionError(
         'Unsupported preupgrade version {} provided for '
-        'entity variants existence tests'.format(from_ver))
+        'entity variants existence tests'.format(_FROM_VERSION))
 
-if to_ver not in supported_ver:
+if _TO_VER not in _supported_ver:
     raise VersionError(
         'Unsupported postupgrade version {} provided for '
-        'entity variants existence tests'.format(to_ver))
-
-
-# The dictionary of entity variants where the key is a component name
-# and value is list of all the component entity variants list those changes
-# during satellite versions
-#
-# Structure of variants directory:
-# {'component_name_e.g_filter,repository':
-#        [
-#        [variant1_6.1, variant1_6.2],
-#        [variant2_6.1, variant2_6.2],
-#        ]
-# }
-#
-# Note: The variants should be listed from 6.1 onwards
-entity_varients = {
-    'filter': [
-        ['lookupkey', 'variablelookupkey'],
-        ['(miscellaneous)', 'foremanopenscap::arfreport'],
-        ['organization', 'katello::subscription'],
-        ['configtemplate', 'provisioningtemplate'],
-        ['view_templates, create_templates, edit_templates, '
-         'destroy_templates, deploy_templates',
-         'view_provisioning_templates, create_provisioning_templates, '
-         'edit_provisioning_templates, destroy_provisioning_templates, '
-         'deploy_provisioning_templates']
-    ]
+        'entity variants existence tests'.format(_TO_VER))
+
+# dict where key is component and value is a dict of properties which name
+# changed from satellite 6.1 to 6.2.
+# On this last dict the key is the property's name on 6.1 and value the name
+#  on 6.2
+_SAT_6_DOT_1_TO_6_DOT_2 = {
+    'filter': {
+        'lookupkey': 'variablelookupkey',
+        '(miscellaneous)': 'foremanopenscap::arfreport',
+        'organization': 'katello::subscription',
+        'configtemplate': 'provisioningtemplate',
+        'view_templates, create_templates, edit_templates, '
+        'destroy_templates, deploy_templates':
+            'view_provisioning_templates, create_provisioning_templates, '
+            'edit_provisioning_templates, destroy_provisioning_templates, '
+            'deploy_provisioning_templates'
+    }
 }
 
 
-def assert_varients(component, pre, post):
-    """Alternates the result of assert if the value of entity attribute is
-    'expected' to change during upgrade
-
-    It takes help from the entity_varients directory above for known changes
-
-    e.g IF filters resource type 'lookupkey' in 6.1 is expected to change to
-    'variablelookupkey' when upgraded to 6.2, then
-    It returns true to pass the test as its expected
+def property_name_on_post_version(component, pre):
```

Comment:
Completely agree. Naming is most difficult on software engineering. I tried the best based on name vars. So we had the names "component", "entity", so I was not sure how to call this function. So it is mapping a name from one version to another.